src/clojure/puppetlabs/ssl_utils/core.clj

```diff
@@ -1043,3 +1042,19 @@
   [c :- CertOrCSR
    digest :- schema/Str]
   (SSLUtils/getFingerprint c digest))
+
+(schema/defn ^:always-validate prevent-duplicate :- (schema/maybe '(schema/Int))
```

Comment:
I don't think this needs the `maybe` - it'll still be a collection of ints, even if it's empty. `maybe` is only needed when the value might be `nil`.